SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveBaseCampaign.java

```diff
@@ -82,6 +77,9 @@ public SwrveBaseCampaign(SwrveBase<?, ?> controller, JSONObject campaignData) th
         setTalkController(controller);
         SwrveLogger.i(LOG_TAG, "Loading campaign " + getId());
 
+        // Start with an empty state
+        this.state = new SwrveCampaignState();
```

Comment:
Sorry, I'm probably being too pedantic here, so keep it as you wish if you like. 

We now have two objects called SwrveCampaignStatus and SwrveCampaignState. Perhaps the SwrveCampaignStatus enum belong inside SwrveCampaignState?